### ISSUE TITLE
after upgrade ovs-workers were not running on 3th node

### ISSUE THREAD

**Issue by pploegaert** [openvstorage/framework#1028]
ovs-workers.log shows:

```
2016-10-13 14:11:23 87800 +0200 - tenv191-node3 - 17738/139849334425408 - celery/celery.concurrency.asynpool - 3249 - DEBUG - removing tasks from inqueue until task handler finished
libust[105857/105857]: Warning: HOME environment variable not set. Disabling LTTng-UST per-user tracing. (in setup_local_apps() at lttng-ust-comm.c:305)
Traceback (most recent call last):
  File "/opt/OpenvStorage/ovs/celery_run.py", line 140, in <module>
    cache.delete(key)
  File "/opt/OpenvStorage/ovs/extensions/storage/persistent/pyrakoonstore.py", line 102, in delete
    raise KeyNotFoundException(field.message)
ovs.extensions.storage.exceptions.KeyNotFoundException: ovs_ensure_single_ovs.vdisk.dtl_checkup

```

**Comment by khenderick** [comment]
Fixed by #1029, packaged in openvstorage-2.7.4-rev.4130.36eeced

**Comment by JeffreyDevloo** [comment]
### Information

In terms of upgrading and race conditions, a regression test should be implemented instead of manual verification.
Manually verifying a delicate subject such as the upgrade is based on some flawed logic.
Added the regression test to my todo's.

### Steps
- Upgraded a recent ovs cluster

Upgraded from 
- openvstorage                       2.7.4-rev.4161.5f26491-1                                 amd64        openvStorage
- openvstorage-backend               1.7.4-rev.766.0d9f30e-1                                  amd64        openvStorage Backend plugin
- openvstorage-backend-core          1.7.4-rev.766.0d9f30e-1                                  amd64        openvStorage Backend plugin core
- openvstorage-backend-webapps       1.7.4-rev.766.0d9f30e-1                                  amd64        openvStorage Backend plugin Web Applications
- openvstorage-cinder-plugin         1.2.2-rev.38.dcc3b76-1                                   amd64        OpenvStorage Cinder plugin for OpenStack
- openvstorage-core                  2.7.4-rev.4161.5f26491-1                                 amd64        openvStorage core
- openvstorage-hc                    1.7.4-rev.766.0d9f30e-1                                  amd64        openvStorage Backend plugin HyperConverged
- openvstorage-sdm                   1.6.4-rev.390.89e2ad4-1                                  amd64        Open vStorage Backend ASD Manager
- openvstorage-webapps               2.7.4-rev.4161.5f26491-1                                 amd64        openvStorage Web Applications

### Output

Newly installed packages:
- openvstorage                       2.7.4-rev.4161.5f26491-1                                 amd64        openvStorage
- openvstorage-backend               1.7.4-rev.770.ae31d34-1                                  amd64        openvStorage Backend plugin
- openvstorage-backend-core          1.7.4-rev.770.ae31d34-1                                  amd64        openvStorage Backend plugin core
- openvstorage-backend-webapps       1.7.4-rev.770.ae31d34-1                                  amd64        openvStorage Backend plugin Web Applications
- openvstorage-cinder-plugin         1.2.2-rev.38.dcc3b76-1                                   amd64        OpenvStorage Cinder plugin for OpenStack
- openvstorage-core                  2.7.4-rev.4161.5f26491-1                                 amd64        openvStorage core
- openvstorage-hc                    1.7.4-rev.770.ae31d34-1                                  amd64        openvStorage Backend plugin HyperConverged
- openvstorage-sdm                   1.6.4-rev.390.89e2ad4-1                                  amd64        Open vStorage Backend ASD Manager
- openvstorage-webapps               2.7.4-rev.4161.5f26491-1                                 amd64        openvStorage Web Applications

Other conditionals:
- [x] All services are running
- [x] Everything is still healthy
- [x] Process flow works (backend creation, initialize disk, claim asd, add vpool, add vdisk)
- [x] Parse lib.log for potential update errors
  Found no update errors

### Test result

Test passed.

### Packages
- openvstorage                       2.7.4-rev.4161.5f26491-1                                 amd64        openvStorage
- openvstorage-backend               1.7.4-rev.770.ae31d34-1                                  amd64        openvStorage Backend plugin
- openvstorage-backend-core          1.7.4-rev.770.ae31d34-1                                  amd64        openvStorage Backend plugin core
- openvstorage-backend-webapps       1.7.4-rev.770.ae31d34-1                                  amd64        openvStorage Backend plugin Web Applications
- openvstorage-cinder-plugin         1.2.2-rev.38.dcc3b76-1                                   amd64        OpenvStorage Cinder plugin for OpenStack
- openvstorage-core                  2.7.4-rev.4161.5f26491-1                                 amd64        openvStorage core
- openvstorage-hc                    1.7.4-rev.770.ae31d34-1                                  amd64        openvStorage Backend plugin HyperConverged
- openvstorage-sdm                   1.6.4-rev.390.89e2ad4-1                                  amd64        Open vStorage Backend ASD Manager
- openvstorage-webapps               2.7.4-rev.4161.5f26491-1                                 amd64        openvStorage Web Applications
